docs/cloud/cloud-query/cloud-query-execute.md

```diff
@@ -16,10 +16,7 @@ You can also query data programmatically with [the query endpoint](https://api-d
 
 ## Before you begin
 
-{% include /cloud/cloud-before-begin.md %}
-* [Learn how to create a database](/docs/cloud/cloud-databases/cloud-db-create)
-* [Learn how to create a table](/docs/cloud/cloud-tables/cloud-table-create)
-* [Learn about querying in Cloud](/docs/cloud/cloud-query/cloud-query-data)
+{% include /cloud-query/cloud-before-query.md %}
 
 ## Step 1: Navigate to query editor
```

Comment:
This step 1 could easily be:

* Added under "Before you begin"
* Added as a first step under what is now "Step 2".